logger: string overflow warning with gcc7.3

String overflow...

```
gcc -DHAVE_CONFIG_H -I. -I..    -DLOGGER_FORMAT -g -O2 -MT sof_logger-logger_convert.o -MD -MP -MF .deps/sof_logger-logger_convert.Tpo -c -o sof_logger-logger_convert.o `test -f 'logger_convert.c' || echo './'`logger_convert.c
logger_convert.c: In function ‘convert’:
logger_convert.c:120:22: warning: ‘__builtin___sprintf_chk’ may write a terminating nul past the end of the destination [-Wformat-overflow=]
   sprintf(ids, "%d.%d", dma_log.id_0, dma_log.id_1);
                      ^
In file included from /usr/include/stdio.h:862:0,
                 from convert.h:15,
                 from logger_convert.c:23:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:33:10: note: ‘__builtin___sprintf_chk’ output between 4 and 10 bytes into a destination of size 9
   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       __bos (__s), __fmt, __va_arg_pack ());
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
mv -f .deps/sof_logger-logger_convert.Tpo .deps/sof_logger-logger_convert.Po
gcc -DLOGGER_FORMAT -g -O2   -o sof-logger sof_logger-rmbox.o sof_logger-logger_convert.o  
```

--- a/rmbox/logger_convert.c
+++ b/rmbox/logger_convert.c
@@ -27,7 +27,7 @@
 #define TRACE_MAX_PARAMS_COUNT		4
 #define TRACE_MAX_TEXT_LEN		1024
 #define TRACE_MAX_FILENAME_LEN		128
-#define TRACE_MAX_IDS_STR		9
+#define TRACE_MAX_IDS_STR		10
 
 /* logs file signature */
 #define SND_SOF_LOGS_SIG_SIZE	4

